Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptionsResolver.swift

```diff
@@ -73,15 +84,15 @@ open class _ChatMessageLayoutOptionsResolver<ExtraData: ExtraDataTypes> {
         if isLastInSequence && !message.isSentByCurrentUser && !channel.isDirectMessageChannel {
             options.insert(.authorName)
         }
-        if message.quotedMessage?.id != nil {
+        if isQuoted(message) {
             options.insert(.quotedMessage)
         }
```

Comment:
like @evsaev mentioned, this should actually be: 
```suggestion
        if hasQuotedMessage(message) {
            options.insert(.quotedMessage)
        }
```
Because this checks if the current message is quoting a message. Another option is: `isQuoting(message)`.